tests/components/image_processing/test_init.py

```diff
@@ -7,286 +7,256 @@
 from homeassistant.core import callback
 from homeassistant.exceptions import HomeAssistantError
 from homeassistant.loader import DATA_CUSTOM_COMPONENTS
-from homeassistant.setup import setup_component
+from homeassistant.setup import async_setup_component
 
-from tests.common import (
-    assert_setup_component,
-    get_test_home_assistant,
-    get_test_instance_port,
-)
+from tests.common import assert_setup_component, get_test_instance_port
 from tests.components.image_processing import common
 
 
-class TestSetupImageProcessing:
-    """Test class for setup image processing."""
+def get_url(hass):
+    """Return camera url."""
+    state = hass.states.get("camera.demo_camera")
+    return f"{hass.config.internal_url}{state.attributes.get(ATTR_ENTITY_PICTURE)}"
 
-    def setup_method(self):
-        """Set up things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
 
-    def teardown_method(self):
-        """Stop everything that was started."""
-        self.hass.stop()
+async def setup_image_processing(hass):
+    """Set up things to be run when tests are started."""
+    hass.data.pop(DATA_CUSTOM_COMPONENTS)
```

Comment:
I think we can use this fixture instead:
https://github.com/home-assistant/core/blob/5c717cbb1d9a0f9c19da06173520c51cf6ddec47/tests/conftest.py#L619-L622

Reply:
Fixed in 0816a0b